wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -860,6 +860,14 @@ extension ZMUserSession: ZMSyncStateDelegate {
         Task {
             await self.cRLsChecker.checkExpiredCRLs()
         }
+<<<<<<< HEAD
+=======
+
+        managedObjectContext.performGroupedBlock { [weak self] in
+            self?.notifyThirdPartyServices()
```

Comment:
@KaterinaWire I have had removed this call `notifyThirdPartyServices()` and the `performGroupedBlock`, so that's why the merge conflict appeared.

Reply:
Resolved